### PR TITLE
[BUG] Orientation Change on ProfileFragment Fix

### LINKED ISSUE #824: [BUG] Orientation Change on ProfileFragment
On viewing the user's profile .i.e., **ProfileFragment** and on **orientation change** from the same leads to the **chatrooms** and not to the Profile Fragment.

issue_1

### PR DESCRIPTION
**Pull request by SyamSundarKirubakaran**
@RocketChat/android

Closes #824 

pr 2

### CODE REVIEW

**Comment by philipbrito** [comment]
Thanks for your contribution @SyamSundarKirubakaran !
We'll review it late!
Thank you!

**Comment by rafaelks** [comment]
@SyamSundarKirubakaran Orientation is working for me, but the screen doesn't scroll, so I'm not able to see all the fields on my screen completely. Do you mind adding a scroll view on this screen also?

**Comment by SyamSundarKirubakaran** [comment]
@rafaelks I've resolved some merge conflicts and added a scroll view. Sorry for the delayed commit, was a bit held up. Thanks.

File: app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt [hunk 1]
```diff
@@ -42,6 +42,7 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector, HasSupp
     @Inject lateinit var fragmentDispatchingAndroidInjector: DispatchingAndroidInjector<Fragment>
     @Inject lateinit var presenter: MainPresenter
     private var isFragmentAdded: Boolean = false
+    private var CURRENT_STATE: String = "CURRENT_STATE"
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt]: It's supposed to be a `const`, so declare it as such.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by philipbrito** [comment]
Closing this PR since it is already fixed by #1337

**Pull request closed without merging**